E2-Breakdown-Money/src/main/java/com/redelles/e2/Breakdown.java

```diff
@@ -0,0 +1,15 @@
+package com.redelles.e2;
+
+
+
+public class Breakdown {
+
+    public static Integer breakdownWithQuantity(Integer quantity, Integer result){
+
+        Integer breakdownQty = result / quantity;
+        System.out.println(breakdownQty + " -> " + quantity);
```

Comment:
System.out.println() inside the breakdown.
Breakdown is a class with a functionality.
System.out.println() should be a part of the view, therefore, it shouldn't be inside this class.
If you want to use the Breakdown class in other project or place, it always will show on terminal, when is not a behaviour which should be in his context. Because is for a View context